<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QDialog" name="Calculator">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="3">
    <widget class="QPushButton" name="minbtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 121, 0);
font: 14pt &quot;System&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
border-radius: 0px;
color:rgb(255,255,255);
</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="num6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);
font: 30pt &quot;System&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
border-radius: 0px
</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="num2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);
font: 30pt &quot;System&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
border-radius: 0px
</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="num4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);
font: 30pt &quot;System&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
border-radius: 0px
</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="num9">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);
font: 30pt &quot;System&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
border-radius: 0px
</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="num3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);
font: 30pt &quot;System&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
border-radius: 0px
</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="num7">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);
font: 30pt &quot;System&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
border-radius: 0px
</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="clearbtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(136, 138, 133);
font: 14pt &quot;System&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
border-radius: 0px;
color: rgb(255, 255, 255)
</string>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="mulbtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 121, 0);
font: 14pt &quot;System&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
border-radius: 0px;
color:rgb(255,255,255);
</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="num5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);
font: 30pt &quot;System&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
border-radius: 0px
</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="sumbtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 121, 0);
font: 14pt &quot;System&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
border-radius: 0px;
color:rgb(255,255,255);
</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="num1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);
font: 30pt &quot;System&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
border-radius: 0px
</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="autoRepeatInterval">
      <number>107</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QPushButton" name="num0">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);
font: 30pt &quot;System&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
border-radius: 0px
</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="divbtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 121, 0);
font: 14pt &quot;System&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
border-radius: 0px;
color:rgb(255,255,255);
</string>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="num8">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(238, 238, 236);
font: 30pt &quot;System&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
border-radius: 0px
</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QPushButton" name="eqbtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(245, 121, 0);
font: 14pt &quot;System&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
border-radius: 0px;
color: rgb(255, 255, 255)
</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLineEdit" name="textbox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>15</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 134, 166,0);
border-radius: 0px;
color: rgb(255, 255, 255);
font: 700 15pt &quot;Segoe UI&quot;;
font: 700 15pt &quot;Segoe UI&quot;;
</string>
     </property>
     <property name="cursorMoveStyle">
      <enum>Qt::VisualMoveStyle</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
